src/library/Uno.Cupertino/Uno.Cupertino.csproj

```diff
@@ -10,11 +10,31 @@
 
 	<ItemGroup>
 		<PackageReference Include="DotNet.ReproducibleBuilds" Version="0.1.26" PrivateAssets="All" />
+		<PackageReference Include="Uno.XamlMerge.Task" Version="0.1.0-dev.41" />
 	</ItemGroup>
 
 	<ItemGroup>
 		<UnoSourceGeneratorAdditionalProperty Include="UseWinUI" />
 	</ItemGroup>
+
+	<ItemGroup>
+		<XamlMergeInput Include="**\*.xaml" Exclude="bin\**\*.xaml;obj\**\*.xaml" />
+	</ItemGroup>
+	
+	<Choose>
+		<When Condition="'$(TargetFramework)'=='xamarinios10' or '$(TargetFramework)'=='monoandroid10.0' or '$(TargetFramework)'=='monoandroid11.0'">
+			<ItemGroup>
+				<IncludeXamlNamespaces Include="mobile" />
+				<ExcludeXamlNamespaces Include="not_mobile" />
+			</ItemGroup>
+		</When>
+		<Otherwise>
+			<ItemGroup>
+				<IncludeXamlNamespaces Include="not_mobile" />
+				<ExcludeXamlNamespaces Include="mobile" />
+			</ItemGroup>
+		</Otherwise>
+	</Choose>
```

Comment:
Just out of curiosity, would this method be preferred over doing something like IsTypePresent(... NativeFramePresenter) or some other type that exists only for iOS/Android? I guess maybe that could be too fragile since native types may not be in Uno forever?

Reply:
(same answer as above, WinUI does more validation that UWP, so IsTypePresent is not applicable)